src/controllers/dynakube/istio/config.go

```diff
@@ -1,9 +1,23 @@
 package istio
 
 import (
+	"fmt"
+
 	"github.com/Dynatrace/dynatrace-operator/src/logger"
 )
 
 var (
 	log = logger.Factory.GetLogger("dynakube-istio")
 )
+
+const (
+	OperatorComponent = "operator"
+	OneAgentComponent = "oneagent"
+
+	IstioGVRName    = "networking.istio.io"
+	IstioGVRVersion = "v1alpha3"
```

Comment:
do we need to document with which istio version we are compatible?

Reply:
The official docu has examples for both `v1alpha3` and `v1beta1`, so we are safe as far as support, but we should move to v1beta1, but we should look into that separately